backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/AbstractInvoiceBL.java

```diff
@@ -2033,4 +2050,31 @@ public I_C_InvoiceLine getLineById(@NonNull final InvoiceAndLineId invoiceAndLin
 	{
 		return invoiceDAO.retrieveLineById(invoiceAndLineId);
 	}
+
+	@NonNull
+	private BigDecimal getOpenAmtAbs(
```

Comment:
IMHO this name is misleading....

Reply:
better call it something like `getOpenAmt_AP_CM_Adjusted`?